app/src/main/java/no/usn/mob3000/ui/components/DangerousActionDialogue.kt

```diff
@@ -34,9 +33,7 @@ fun DangerousActionDialogue(
     onDismiss: () -> Unit,
     confirmText: String = stringResource(R.string.component_dialogue_dangerous_action_button_confirm),
     dismissText: String = stringResource(R.string.component_dialogue_dangerous_action_button_dismiss),
-    colorContainer: Color = MaterialTheme.colorScheme.surface,
-    colorButtonDismiss: Color = DefaultButton,
-    colorButtonConfirm: Color = Color.Red,
```

Comment:
Same problem as the previous pull request. These are there for a reason, this composable is _meant_ to have colors as input parameters. Here you are to change the default parameter value, _not_ remove them. It makes them less flexible.